src/routes/inngang/inngang-innsendingsvalg-post.tsx

```diff
@@ -64,52 +60,19 @@ const InngangInnsendingPost = ({
             <ContentContainer>
                 <CenteredPageTitle>{title}</CenteredPageTitle>
 
-                <Row>
-                    <DineAnkemuligheter show={showAnkeList} temaKey={temaKey} />
-                </Row>
-
                 <WhiteSection>
                     <CenteredSectionTitle>{inngang.innsendingsvalg.post.title}</CenteredSectionTitle>
+                    <DineAnkemuligheter show={showAnkeList} temaKey={temaKey} />
```

Comment:
Skal vi vise ankemuligheter når det ikke er mulig å klage digitalt?
Mener vi diskuterte dette for litt siden 🤔